dataverse-persistence/src/main/resources/db/migration/V23__addGrantSuggestion.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE IF NOT EXISTS grantSuggestions
+(
+    id                 SERIAL       NOT NULL,
+    grantAgency        varchar(255) NOT NULL,
+    grantAgencyAcronym varchar(255) NOT NULL,
+    fundingProgram     varchar(255) ,
```

Comment:
We have to revert V23 to the state it's already on develop. Add this change to v25.